src/Pdf/Engine/WkHtmlToPdfEngine.php

```diff
@@ -164,6 +157,54 @@ protected function _getCommand(): string
         return $command;
     }
 
+    /**
+     * Parses a value of options to create a part of the command.
+     * Created to reuse logic to parse the cover and toc options.
+     *
+     * @param string $key the option key name
+     * @param string|bool|array $value the option value
```

Comment:
```suggestion
     * @param string|true|array $value the option value
```
Let's be extra accurate here :)